oscarlab/graphene
# Documentation fixups

## Description of the changes 

Various fixups for our docs. See commit messages for details.

## How to test this PR? 

Build and check documentation.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/1422)
